kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -4207,4 +3979,68 @@ public void onFinish() {
         typingTimerMap.put(userId, timer);
         return timer;
     }
+
+    @Override
+    public void onRecordStart() {
+        vibrate();
+        isRecording = true;
+        if (recordButton != null && getContext() != null) {
```

Comment:
```
  if (recordButton != null && getContext() != null) {
            KmUtils.setBackground(getContext(), recordButton, R.drawable.km_audio_button_background);
        }

 if (messageEditText != null) {
            messageEditText.setVisibility(View.GONE);
        }

```
move above code in in another  funcation  as this is used in `onRecordFinish  `, `onRecordStart` and `onLessThanSecond`